### DISCUSSION TITLE
Section and subsection headings too small in PDF (scrartcl class), headings=big ignored

### DISCUSSION THREAD

**Discussion by ElDuderino321** [jgm/pandoc#10135] [Q&A] [answered]
I've created a syllabus in Markdown, with the following variables in the frontmatter:

```
documentclass: scrartcl
classoption:
    - 12pt
    - ngerman    
    - american
    - headings=big
geometry:
    - verbose
    - lmargin=1.9cm
    - rmargin=1.9cm
    - tmargin=2.9cm
mainfont: Garamond Libre
sansfont: Open Sans
header-includes: |
    \usepackage{fancyhdr}
    \pagestyle{fancy}
    \usepackage{graphicx} 
    \renewcommand{\maketitle}{}
```

But the size for level 1,2, and 3 headings look like it's just the regular font size in bold. They are not standing out. Maybe level 1 heading is 1 pt bigger.

Strangely, in a previous version created in LyX, using the same document class and other settings, the headings have the right sizes (and that is even without the headings=big setting set in the class options).

Does pandoc's Latex template have something to do with this? How can I get the font size right?

**Comment by TomBener** [comment, marked as answer]
I've tried your example and it seems the font size of headings is not incorrect.

![CleanShot 2024-09-01 at 08 11 06@2x](https://github.com/user-attachments/assets/400e2d9c-21ba-49b3-bc7e-a58c5dd91440)

If you want to change the font size of headings, you can try [these methods](https://tex.stackexchange.com/questions/59726/change-size-of-section-subsection-subsubsection-paragraph-and-subparagraph-ti).

**Reply by ElDuderino321** [reply]
Thanks! I got it the way I wanted it using

```
\setkomafont{section}{\Huge}
\setkomafont{subsection}{\huge}
\setkomafont{subsubsection}{\Large}
```

I played with different setting, removing setkomafont and using different headings= setting, and I'm wondering if either my previous Latex setup was different (but it's the same install, and same options) and has unusually "large" headings, or something other I don't understand. But if I don't use setkomafont, and only headings=big, it looks like this:

![image](https://github.com/user-attachments/assets/915faecb-f603-4b40-b48d-128542dc5689)

To me, that's barely standing out. In my old version (made in LyX), it looked like this:

![image](https://github.com/user-attachments/assets/7879f18c-e493-44f5-bd02-ed57139cf9a1)

Now, with setkomafont used like above, I got it looking like this:

![image](https://github.com/user-attachments/assets/0c5cd5e5-3774-4528-a446-e8a177b4586a)

Are my expectations for heading size unusually "huge," or what could be going on? I checked all the relevant settings, both documents should be produced with the same class and class settings.

(But that's just me being curious, my problem got solved, thank you!!)

**Comment by jpcirrus** [comment]
The KOMA-Script default for the `headings` class option is `big`, which according to p. 104 of `scrguide-en` applies the following default values to the `scrbook` and `scrreprt` classes:

| **Element**   | **Value**             |
|---------------|-----------------------|
| part          | \Huge                 |
| partnumber    | \huge                 |
| chapter       | \huge                 |
| chapterprefix | \usekomafont{chapter} |
| section       | \Large                |
| subsection    | \large                |
| subsubsection | \normalsize           |
| paragraph     | \normalsize           |
| subparagraph  | \normalsize           |

 Of note:

> `scrartcl` generally uses slightly smaller headings. (p. 96)

Obviously, your, and mine too when initially using the KOMA classes, taste differs. I now defer to their typographic expertise.

**Reply by ElDuderino321** [reply]
Lol, I hear you! Thanks for clearing that up for me. I wonder what it was about my previous setup that made the fonts so big, even though I'm sure it's the same settings. Maybe I'll find out, and I'll play around with more settings a bit. Thanks!
